python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -82,6 +82,10 @@ def fit(self, data, epochs=1, batch_size=32):
 
         # fit on internal
         if self.distributed:
+            if batch_size % self.workers_per_node != 0:
+                raise RuntimeError("Please make sure that 'batch_size' "
```

Comment:
In the error message, print the value of `batch_size` and `worker_per_node`.

Reply:
Okay, I will add relevant values ​​in the next submission